taipy/gui/viselements.json

```
@@ -133,6 +133,18 @@
                         "type": "str",
                         "default_value": "None",
                         "doc": "The label associated with the input."
+                    },
+                    {
+                        "name": "step",
+                        "type": "int|float",
+                        "default_value": "1",
+                        "doc": "The amount that the value changes on each increment or decrement."
+                    },
+                    {
+                        "name": "step_multiplier",
+                        "type": "int|float",
+                        "default_value": "10",
+                        "doc": "The amount that the value changes on each increment or decrement when the shift key is pressed."
```

I think the doc is wrong. If it's not, the code is (which I think it is not):
```suggestion
                        "doc": "A factor that multiplies <i>step</i> when the user presses the Shift key while clicking one of the arrow buttons ."
```

---

Updated